From 75be8a16e3686b306fe51eb72a67ef9cad3067dc Mon Sep 17 00:00:00 2001
From: Eric Salama <esalama@haproxy.com>
Date: Tue, 8 Jan 2019 10:49:57 +0100
Subject: [PATCH] package: flx: include <sys/sysmacros.h> explicitely. This is
 required since glibc 2.28

---
 fct1.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/flx/fct1.c b/flx/fct1.c
index b0c9e5e..b656b53 100644
--- a/flx/fct1.c
+++ b/flx/fct1.c
@@ -12,6 +12,7 @@
 #include <string.h>
 #include <time.h>
 #include <stdlib.h>
+#include <sys/sysmacros.h>
 
 #include "flx.h"
 #include "utils.h"
-- 
2.20.1

